Source/Scene/ShadowMap.js

```diff
@@ -791,7 +1055,20 @@ define([
         }
     };
 
+    ShadowMap.prototype.updatePass = function(context, pass) {
+        if (this._isPointLight && this._usesCubeMap) {
+            this._framebuffer._attachTexture(context, WebGLConstants.COLOR_ATTACHMENT0, this._passFaces[pass]);
```

Comment:
I can try that out and see. The one disadvantage is each framebuffer needs its own depth texture (in addition to the shadow texture).